traffic_control/clients/python/trafficops/__version__.py

```diff
@@ -14,4 +12,4 @@
 # limitations under the License.
 #
 
-__version__ = '1.1.1'
+__version__ = '1.1.1-1'
```

Comment:
This kind of goes back to the versioning discussion we were having on your other PR. If we're going to be following some kind of versioning practice it should probably be documented in a `VERSIONING.md` file in the `clients/python` directory. For instance, I assume you're adding `-1` since it's not really part of the version and wouldn't cause clients already installed at 1.1.1 to upgrade to this version?

Reply:
I think `pip` would still do the upgrade if you asked it to. Not totally sure. I added a patch number because this change didn't add functionality or really change anything, it's just a bugfix. You think a micro version is more appropriate? I'm not totally clear on that to be honest.

As far as documenting the versioning scheme, I can certainly get behind that but i'd prefer it to live in our main docs rather than a separate MarkDown file. My suggestion would be to include it in the docstring of the `__version__` module, or maybe in the package docstring. That way, sphinx-autodoc will pick it up and put it in the main docs, and it will live with the package wherever it's hosted and however it's installed.

Reply:
Bugfixes should be `patch` in the `major.minor.patch` scheme if we're choosing to use SemVer for the client versioning scheme. I'm okay with describing the versioning scheme in a docstring, I'd just be a little concerned that people wouldn't easily notice it. A `VERSIONING.md` file would clearly stick out in a directory listing, and people generally look for things like a `README.md` when learning about a new project.

Reply:
I added a documented versioning strategy, hope it makes sense.